lib/everypolitician/popolo/event.rb

```diff
@@ -1,6 +1,6 @@
 module Everypolitician
   module Popolo
-    class Event < Entity
+    class Event < EntityWithDynamicMethods
```

Comment:
https://github.com/everypolitician/everypolitician-popolo/issues/56 should mean that Event doesn't need to go through this indirection — the work on that class has already been done.

It does, however, also raise the question as to why this approach is needed for some classes yet wasn't there. I don't quite understand what's going on here well enough yet to know understand that, I'm afraid.